src/ray/object_manager/pull_manager.cc

```diff
@@ -25,20 +25,29 @@ PullManager::PullManager(
 uint64_t PullManager::Pull(const std::vector<rpc::ObjectReference> &object_ref_bundle,
                            bool is_worker_request,
                            std::vector<rpc::ObjectReference> *objects_to_locate) {
+  // To avoid edge cases dealing with duplicated object ids in the bundle,
```

Comment:
This also fixes an issue where we failed to issue a pull since num_objects_missing_sizes was calculated incorrectly with duplicate object ids.